consensus/spos/constants.go

```diff
@@ -23,4 +23,4 @@ const ValidatorPeerHonestyDecreaseFactor = -2
 
 // MaxNumOfMessageTypeAccepted represents the maximum number of the same message type accepted in one round to be
 // received from the same public key
-const MaxNumOfMessageTypeAccepted = 1
+const MaxNumOfMessageTypeAccepted = 2
```

Comment:
can we change the num of message types only for signatures?
not for the other messages

Reply:
done